cobalt/doc/versioning.md

```diff
@@ -2,15 +2,19 @@
 
 Cobalt versions, as they appear in the user agent, have the following structure:
 
-**[Feature Year]**.**[Purpose]**.**[Update Number]**.**[Build ID]**
+**[Feature Year]**.**[Purpose]**.**[Update Number]**
 
 The meansions of these components are described below.
```

Comment:
I think there is a typo with "meansions". Let's change this to:
"The definitions of these version structure components are described below."

Reply:
Thank you. It is fixed